binplace.targets

```diff
@@ -35,6 +35,9 @@
       <TargetingPackDir Include="$(RefPath)" />
       <TargetingPackDir Condition="'$(IsNETCoreAppRef)' == 'true'" Include="$(BinDir)netcoreapp\pkg\ref" />
     </ItemGroup>
+    <ItemGroup Condition="'$(NuGetTargetMoniker)' == '' Or $(NuGetTargetMoniker.Contains('UAP'))">
```

Comment:
`'$(NuGetTargetMoniker)' == ''` part of the condition was actually copy/pasted by mistake, the only condition that should be there is the UAP one. @weshaggard RefPath is not set to UAP since in dir.props it gets initialized to netcoreapp, so perhaps the change has to go there instead. I'll remove this small change on the file and try to figure out a better way to calculate this.